compiler/noirc_frontend/src/hir/comptime/errors.rs

```diff
@@ -1,11 +1,11 @@
-use crate::Type;
+use crate::{hir::def_collector::dc_crate::CompilationError, Type};
 use acvm::FieldElement;
-use noirc_errors::Location;
+use noirc_errors::{CustomDiagnostic, Location};
 
 use super::value::Value;
 
 /// The possible errors that can halt the interpreter.
-#[derive(Debug)]
+#[derive(Debug, Clone)]
```

Comment:
```suggestion
#[derive(Debug)]
```

Reply:
Don't think this is used?

Reply:
Looks like it is required by the bound on CompilationError

Reply:
Ah my bad I missed that